<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox::drop-down 
{
    border: 0px;
}

QComboBox::down-arrow {
    image: url(&quot;:/images/scrol.png&quot;);
    width: 20px;
    height: 20px;
}

  QComboBox:pressed {
 	border: 2px solid rgb(159, 122, 16);
	background: rgb(248, 248, 128);
}
QComboBox{border: 1px solid gray;
     border-radius: 3px;
     padding: 1px 18px 1px 3px;
     min-width: 6em;
 }
 QComboBox:editable {
     background: white;

 }



QTabBar::tab {
font: bold;
  background:  rgb(248, 248, 128);
color: grey;

   border: 2px solid rgb(159, 122, 16);
  width: 112px;
height: 20px; 
 }
QTabBar::tab:selected {
  color: black;
 }

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(248, 248, 128);

    border-top: 2px solid rgb(100, 2, 24);
  border-right: 6px outset rgb(159, 122, 16);
  border-bottom: 6px outset rgb(159, 122, 16);
  border-left: 6px outset rgb(159, 122, 16);

 }

 QPushButton:hover {
background-color: rgb(248, 238, 0);
  border-right: 6px outset rgb(200, 122, 16);
  border-bottom: 6px outset rgb(200, 122, 16);
  border-left: 6px outset rgb(200, 122, 16);
}</string>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>351</width>
      <height>269</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>1</width>
      <height>0</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel  { font-size: 18px }

QFrame  { 
background-color: rgb(232, 232, 232);
 }


QRadioButton  { font-size: 18px }

 QLineEdit:hover {

  border: 3px solid rgb(248, 248, 128);
}

 QLineEdit:pressed {

  border: 3px solid rgb(248, 248, 128);
background: rgb(232, 232, 232);
}

</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tabChangePin">
     <attribute name="title">
      <string>Сменить PIN</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>281</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Введите старый PIN-код</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editOldPin">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>8</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Введите новый PIN-код</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editPin1">
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Подтвердите новый PIN-код</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editPin2">
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFormat">
     <attribute name="title">
      <string>Форматировать</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Режим :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Без формирования PUK</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>С формированием PUK</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabUnblock">
     <attribute name="title">
      <string>Разблокировать</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>281</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Введите PUK-код</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editPuk">
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Введите новый PIN-код</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editPin1_2">
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Подтвердите новый PIN-код</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editPin2_2">
         <property name="maxLength">
          <number>32</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Устройство №</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBoxDevice">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(248, 248, 128);

    border-top: 2px solid rgb(100, 2, 24);
  border-right: 6px outset rgb(159, 122, 16);
  border-bottom: 6px outset rgb(159, 122, 16);
  border-left: 6px outset rgb(159, 122, 16);

 }

 QPushButton:hover {
background-color: rgb(248, 238, 0);
  border-right: 6px outset rgb(200, 122, 16);
  border-bottom: 6px outset rgb(200, 122, 16);
  border-left: 6px outset rgb(200, 122, 16);
}</string>
    </property>
    <property name="text">
     <string>ОK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>5</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background:url(&quot;:/images/close.png&quot;);
  }

 QPushButton:hover {
 background-image:url(&quot;:/images/close1.png&quot;);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinim">
    <property name="geometry">
     <rect>
      <x>383</x>
      <y>5</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-image:url(&quot;:/images/minim.png&quot;);

    }
 QPushButton:hover {
 background-image:url(&quot;:/images/minim1.png&quot;);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>7</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgb(158, 70, 14);
}</string>
    </property>
    <property name="text">
     <string>СКЗИ ОКБ САПР. Инициализация и форматирование</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
